src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import { useForm } from "react-hook-form";
 import { useFirebase } from "react-redux-firebase";
-import axios from "axios";
-import { codeCheckUrl } from "utils/url";
+import { default as _firebase } from "firebase/app";
```

Comment:
What do we gain from doing this that we can't get from the `useFirebase` hook?

Why are we renaming the import of `firebase` as `_firebase`?

Reply:
There is already a variable named `firebase` which is using the hook `useFirebase` and since that hook doesn't have functions, we need to import the default `firebase` from `firebase/app` to use and functions. That's why this is imported this way, to avoid the name collision.

Reply:
Ah, that functions thing again. Fair enough, makes sense :)

This pattern will get solved properly when we refactor all of the firebase function call/update/etc logic into `api/` (see https://github.com/sparkletown/internal-sparkle-issues/issues/197)